File: 2148/2148.go
func countElementsWithStrictlySmall(root *TreeNode) int {
	if root == nil {
		return 0
	}
	l := countElementsWithStrictlySmall(root.Left)
	r := countElementsWithStrictlySmall(root.Right)
	if l > r {
		return 1 + l
	}
	return 1 + r
}

File: 2148/2148.c
int countElementsWithStrictlySmall(struct TreeNode* root) {
    if (!root) return 0;
    int l = countElementsWithStrictlySmall(root->left);
    int r = countElementsWithStrictlySmall(root->right);
    return 1 + (l > r ? l : r);
}